JhipsterXamarin/UnitTestJhispterXamarin/Tests/UserEntityViewModelTest.cs

```diff
@@ -0,0 +1,96 @@
+﻿using FluentAssertions;
+using JhipsterXamarin.Models;
+using JhipsterXamarin.Services;
+using JhipsterXamarin.ViewModels;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using MvvmCross.Navigation;
+using MvvmCross.Tests;
+
+namespace UnitTestJhispterXamarin
+{
+    [TestClass]
+    public class UserEntityViewModelTest : MvxIoCSupportingTest
+    {
+        private UserEntityViewModel userEntityViewModel;
+
+        private UserModel admin;
+        private UserModel paul;
+        private UserModel john;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            base.Setup(); // from MvxIoCSupportingTest
+
+            var mockUserEntityService = new Mock<IUserEntityService<UserModel>>();
+            var mockNavLoginService = new Mock<IMvxNavigationService>();
+
+            userEntityViewModel = new UserEntityViewModel(mockNavLoginService.Object, mockUserEntityService.Object);
+
+            Ioc.RegisterSingleton<IUserEntityService<UserModel>>(mockUserEntityService.Object);
+            Ioc.RegisterSingleton<IMvxNavigationService>(mockNavLoginService.Object);
+        }
+
+        protected override void AdditionalSetup()
+        {
+            admin = new UserModel();
+            admin.Id = "admin";
```

Comment:
This do not compile. Can you fix it ? We also need to add UT in build step of the github action